docs/application/native/guides/app-management/app-controls.md

```diff
@@ -109,9 +109,9 @@ app_control_destroy(app_control);
 ```
 > **Note**
 >
-> Two APIs that can obtain a reply result were added to return the result.
-> - `app_control_send_launch_request_async()` (Since Tizen 5.0) : Asynchronously launches application controls.
-> - `app_control_send_launch_request_sync()` (Since Tizen 5.5) : Synchronously launches application controls.
+> The following APIs that obtain a reply result were added to return the result.
```

Comment:
@hjhun  you have missed the colon **:**  at the end of the statement.
Kindly incorporate that too
Copy paste the following to avoid missing colon:

The following APIs that obtain a reply as a result were added to return the result:

change . -> :
and **as** after reply word in the statement.

Reply:
done